app/src/main/res/layout/movie_list_fragment.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
hmm, I actually can't find any reference to it.
Feels to me that you should use the simplest layout that satisfy your design requirements (i.e. not use a `RelativeLayout` if you all you need is to center a single view) but it's just a hunch, can't really back it up with proof :)